redirect/routes.php

```diff
@@ -260,6 +260,7 @@ class Route {
         'latex' =>  'sharelatex',
         'tex' =>  'sharelatex',
         'netz' => 'grnvs',
+        'la' => 'linalg',
```

Comment:
You need to switch those two around: `'linalg' => 'la',`

Reply:
i think, with the other changed line, it will makes sense 😉

Reply:
No, your build/tests fail, just pointed out the reason why. Trust me :wink:

Reply:
ah, ok.
then the arrow makes more sense 😅